CMakeLists.txt

```diff
@@ -57,3 +58,36 @@ install(
 )
 # Install a Catkin 'package.xml' file. This is required by REP-136.
 install(FILES package.xml DESTINATION share/${PROJECT_NAME})
+
+##########################
+## Build binary package ##
+##########################
+if(BUILD_PACKAGE)
+  set(DESCRIPTION "An Eigen-based, light-weight C++ Interface to Nonlinear Programming Solvers (Ipopt, Snopt)")
+  set(LINUX_DEPENDS "libeigen3-dev")
+  if(BUILD_IPOPT)
+    list(APPEND LINUX_DEPENDS "coinor-libipopt-dev")
+  endif()
+
+  set(CPACK_PACKAGE_VENDOR "ETHZ-ADRL")
+  set(CPACK_RESOURCE_FILE_LICENSE ${CMAKE_CURRENT_LIST_DIR}/LICENSE)
+  set(CPACK_RESOURCE_FILE_README ${CMAKE_CURRENT_LIST_DIR}/README.md)
+  if(UNIX)
+    set(CPACK_GENERATOR "DEB;TXZ")
+
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+      set(DEB_ARCH "amd64")
+    else()
+      set(DEB_ARCH ${CMAKE_SYSTEM_PROCESSOR})
+    endif()
+
+    set(CPACK_PACKAGE_FILE_NAME "${ARG_PACKAGE_PREFIX}${PROJECT_NAME}_${DEB_ARCH}_linux_${VERSION}")
```

Comment:
I think you can remove the custom VERSION variable and just use the one provided by cmake outlined [here](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_VERSION.html).